airflow/example_dags/example_outlet_event_extra.py

```diff
@@ -69,8 +69,8 @@ def dataset_with_extra_by_context(*, outlet_events=None):
     tags=["produces"],
 ):
 
-    def _dataset_with_extra_from_classic_operator_post_execute(context):
-        context["outlet_events"].extra = {"hi": "bye"}
+    def _dataset_with_extra_from_classic_operator_post_execute(context, results):
```

Comment:
I think this should be `result`?

https://github.com/apache/airflow/blob/4376000eb0589fe6b93aa17f1d0eee4847669517/airflow/models/taskinstance.py#L3015

Reply:
@uranusjr It was working with `results` as well. I have updated to `result`